src/cogs/rules.py

```diff
@@ -20,8 +20,7 @@ def __init__(self, bot):
         self._recent_joins = []
 
         self.massjoin_detect = True
-
-        self.next_massjoin_notif = time.time()
+        self.massjoin_active = False
```

Comment:
what's the difference?

Reply:
The old code used to wait a certain amount after it detected a massjoin and only call out a massjoin again after the timeout was finished.
The new code blocks by waiting for reactions and after that everything gets reset.

The new flag also determines behavior specific to after a massjoin has been the detected, like not removing old entries in the recent_joins list etc.